docs/stencil-docs/reference-docs/front-matter-reference.md

```diff
@@ -256,7 +256,7 @@ gql: "query productById($productId: Int!) {
 }"
 ```
   
-We suggest testing GraphQL queries using the [storefront API playground](https://developer.bigcommerce.com/graphql-playground) to refine them before adding them to your template. You can launch the playground in the context of your store by clicking the **Storefront API Playground** link under the **Advanced Settings** menu in your store's control panel.
+We suggest testing GraphQL queries using the [storefront API playground](https://developer.bigcommerce.com/graphql-playground) to refine them before adding them to your template. If your query contains a double quote `"`, replace it with single quotes `'` or escape the double-quotes `\"`. You can launch the playground from your store by going to **Settings** >  **API** > **Storefront API Playground** in your store's control panel.

```

Comment:
In our docs thus far, we don't use the plural/possessive for the control panel; eg, `store control panel`

```suggestion
We suggest testing GraphQL queries using the [storefront API playground](https://developer.bigcommerce.com/graphql-playground) to refine them before adding them to your template. If your query contains a double quote `"`, replace it with single quotes `'` or escape the double-quotes `\"`. You can launch the playground from your store by going to **Settings** >  **API** > **Storefront API Playground** in your store control panel.

```